t/25-cache-service.t

```diff
@@ -366,7 +400,6 @@ subtest 'Asset download with default usage' => sub {
 };
 
 # The following tests start their own workers
```

Comment:
All tests now start their own workers, right? So this comment could be removed as well.